domains/s1ow.json

```
@@ -0,0 +1,21 @@
+{
+    "description": "! s1owmotion.0 profile page",
+    "repository": "https://github.com/H4iPh0ngM1enB4c/Web",
+    "owner": {
+        "username": "H4iPh0ngM1enB4c",
+        "email": "noobdownload2@gmail.com",
+        "discord": "1027559943462727740"
+    },
+    "record": {
+        "A": [
+          "185.199.108.153",
+          "185.199.110.153",
+          "185.199.109.153"
```

You forgot one A record.
```suggestion
          "185.199.108.153",
          "185.199.110.153",
          "185.199.109.153",
          "185.199.111.153"
```